src/components/sandbox-menu/sandbox-menu.tsx

```diff
@@ -48,13 +49,26 @@ export interface SandboxArgs {
 }
 
 function saveConfig(baseUrl: string, config: Record<string, unknown>) {
+  ConsoleLogger.debug(`sandbox-menu: saveConfig: ${baseUrl}`, config);
```

Comment:
Intention question:  
This log will be a debug log in the browser console and not placed in the file log. This is intended? (Simple yes/no is enough)